Packages/Sites/Sfi.Kateheo/Resources/Private/Templates/Pages/tagsList.html

```diff
@@ -0,0 +1,9 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<ul class="tagsList">
+    <f:for each="{items}" as="item">
```

Comment:
ups.
Wrong place. This file should be stored in TypoScriptObjects folder.

By the way. Maybe it's better to use this file (in case if I'll continue with Menu object:
Private/Templates/TypoScriptObjects/2levelMenu.html